django-app/newstream/settings/base.py

```diff
@@ -30,6 +30,7 @@
     INIT_LOCALSTRIPE=(bool, False),
 
     PAYPAL_API_BASE=(str, ""),
+    PAYPAL_PAYMENT_FAILURE_THRESHOLD=(int, None),
```

Comment:
```suggestion
    PAYPAL_PAYMENT_FAILURE_THRESHOLD=(int, 3),
```
I think it's better we define the default value here